demos/human_pose_estimation_demo/src/human_pose_estimator.cpp

```diff
@@ -42,12 +68,23 @@ HumanPoseEstimator::HumanPoseEstimator(const std::string& modelPath,
     network = netReader.getNetwork();
     InferenceEngine::InputInfo::Ptr inputInfo = network.getInputsInfo().begin()->second;
     inputLayerSize = cv::Size(inputInfo->getTensorDesc().getDims()[3], inputInfo->getTensorDesc().getDims()[2]);
-    inputInfo->setPrecision(InferenceEngine::Precision::U8);
```

Comment:
Please try to keep U8 input due it's more efficient.

Reply:
Done